src/js/actions/ChaosActions.js

```diff
@@ -0,0 +1,103 @@
+var EventEmitter = require("events").EventEmitter;
+var lazy = require("lazy.js");
+var request = require("../helpers/qajaxWrapper");
+
+var config = require("../config/config");
+
+var AppsStore = require("../stores/AppsStore");
+var ChaosEvents = require("../events/ChaosEvents");
+
+function requestApp(appId, onSuccess, onError) {
+  request({
+    url: config.apiURL + "v2/apps/" + appId
+  })
+  .success(function (data) {
+    onSuccess(data.body.app);
+  })
+  .error(function () {
+    onError();
+  });
+}
+
+function deleteTasks(taskIds = [], onSuccess) {
+  request({
+    method: "POST",
+    data: {
+      "ids": taskIds
+    },
+    headers: {
+      "Content-Type": "application/json"
+    },
+    url: config.apiURL + "v2/tasks/delete"
+  })
+  .success(function () {
+    onSuccess();
+  })
+  .error(function (error) {
+    onSuccess();
+  });
```

Comment:
There's two different styles here when using method chaining.
In terms of spacing here we start the call to the next method with the same left spacing as line #23 where `request` first gets called. But a few lines below when using `lazy` the chained methods are indented deeper than where the call to `lazy` occurs.